### ISSUE TITLE
Fix Koreanbuilds

### ISSUE THREAD

**Issue by Soundofdarkness** [Soundofdarkness/RuneBook#11]
Seems like they have a full json api now, not sure

**Comment by TinyRaindrop** [comment]
Why not just remove it? [Ezreal](https://koreanbuilds.net/champion/Ezreal?position=bot) has 40 games, the site is dead.

**Comment by Soundofdarkness** [comment]
Well, I'm pretty sure they only use KR-Master+ for their data, which would mean that 40 games are okay-ish I think, but we can totally remove it too, both options work.

**Comment by TinyRaindrop** [comment]
I think we can keep it, as author [shared the data](https://github.com/Soundofdarkness/RuneBook/issues/2#issuecomment-633248022)

**Comment by Rerago** [comment]
I pulled the data once. It is 1:1 the same data as on the web page.

The question is just how useful the site is. Most champs have roughly 10-15 games. I have no idea but I'd be stumped for removal.

**Comment by Rerago** [comment]
The question is how to deal with the site. It keeps data but not general. But always only builds from HighElo players from Kora (btw. there are also europeanbuilds). But that's why the number of games are so small.

But let's take Lucian as an example. What should we show? He is played on 3 lanes. From each lane the build with the most recent score? Or the top 3 per lane?

**Comment by Soundofdarkness** [comment]
I would guess the most recent per lane for the time being, since I feel like it matters more to have relatively recent scores than diversity, since we already have more than enough diversity through the amount of different sites.

**Comment by Rerago** [comment]
@TinyRaindrop 
What are your thoughts on the subject?

**Comment by TinyRaindrop** [comment]
During champ select I wouldn't trust Koreanbuilds' runes and would immediately open another tab. It only shows runes that **one** player (can be selected on the left, under the patch) picked during his 7 games, which, I think, goes against RuneBook's spirit of providing meaningful information. 

'High elo' rune meta can be much better represented by Lolalytics or U.gg master+ filtering, if there's a need of such functionality.

**Comment by Soundofdarkness** [comment]
Yeah, I mostly agree with this, but I would expect that quite a few would trust a single player more than general stats. Main question here might be if we want to be opinionated or not.

**Comment by Rerago** [comment]
So I have completely rewritten the "koreanbuilds" plugin. Based on the API also used on the site which is a lot more performant. BUT they use an authorization-token. We could pull it out of bundle.js via regex. But that's not really the fine English way and I would never want to publish something like that (unless the owner has no problem with it).

Hence the question. Do we want to ask the owner if he gives us a key or if he is satisfied that we get it automatically?

I could also transfer "RuneBook v1.1.1" or something like that via header (user agent). So he knows where the request comes from :D

But otherwise I am very happy with the plugin. I would only disable the "bookmarking". Since the builds are based on players, that could not be renewed.

PS: We should disable or remove the "bookmark" icon when "bookmarks" is set to false in the plugin. Because currently runebook doesn't care about that at all.

**Comment by Soundofdarkness** [comment]
I'd say we should ask the owner and disable bookmarks at least yep, and generally I feel like offering options is probably a good thing. Based on [https://koreanbuilds.net/faq](https://koreanbuilds.net/faq) I would presume we would get a positive answer but we should totally still do that for good measure. Do you want to do this or should I try to (which I can do, but I'm not exactly looking at my email often so it might take longer than it needs to)

**Comment by Rerago** [comment]
Oh it's ok if you ask or @TinyRaindrop.

I don't really use koreanbuilds. But I wanted to update the plugin to the latest version. And additionally it would be possible for me to support europeanbuilds because only the URL is different.

So we could make a "KR/EU builds". Whether as one plugin or as two would not really matter.

But anyway someone would have to ask :D

**Comment by Soundofdarkness** [comment]
Well, since I don't use it either (in fact, i've only learned about the site because of runebook), i'd say somebody who actively uses it should ask the creator, since they would actually have an argument. Since so far we have 3 people not really using it, so it would look like its not really worth using time on atm. So maybe we just leave the issue open and see if somebody chimes in because they would use the plugin ?
